src/templates/en/2020/stories/user_experience.html

```diff
@@ -0,0 +1,3918 @@
+            data-amp-original-style="position:absolute;pointer-events:none;left:17.23301%;top:15.04854%;width:68.93204%;height:8.09061%;opacity:1"
+            class="amp-wp-121ce43">
+            <div id="el-bda9c706-97e0-4df3-b696-1e05de4ef0f3"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0"
+                 class="amp-wp-a1b70b8"><p class="fill amp-wp-678fd6d"
+                                           data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:1.9014084507042248% 0;font-family:'Poppins',sans-serif;font-size:0.582524em;line-height:1.1;text-align:initial;padding:0% 0%;color:#000000">
+              <span data-amp-original-style="font-weight: 700; color: #fcfcfc" class="amp-wp-d77b52b">Web Almanac</span>
+            </p></div>
+          </div>
+          <div
+            data-amp-original-style="position:absolute;pointer-events:none;left:17.23301%;top:18.4466%;width:65.29126%;height:24.27184%;opacity:1"
+            class="amp-wp-6843b89">
+            <div id="el-648df06e-9808-42bc-8180-ad5f9da54443"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0"
+                 class="amp-wp-a1b70b8"><p class="fill amp-wp-4985cf1"
+                                           data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:6.0223048327137505% 0;font-family:'Poppins',sans-serif;font-size:1.747573em;line-height:1.1;text-align:initial;padding:0% 0%;color:#000000">
+              <span data-amp-original-style="font-weight: 700; color: #f7f779" class="amp-wp-fb8f475">2020</span></p>
+            </div>
+          </div>
+          <div
+            data-amp-original-style="position:absolute;pointer-events:none;left:17.71845%;top:38.67314%;width:68.68932%;height:10.84142%;opacity:1"
+            class="amp-wp-43405b9">
+            <div id="el-cdb28853-0908-4d31-a1ed-10a6fb4954f4"
+                 data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0"
+                 class="amp-wp-a1b70b8"><p class="fill amp-wp-ae8dc76"
+                                           data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:-1.325088339222615% 0;font-family:'Lato',sans-serif;font-size:0.404531em;line-height:1.5;text-align:initial;padding:0% 0%;color:#000000">
+              <span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81">HTTP Archive's annual</span>
+              <span data-amp-original-style="font-weight: 900; color: #fcfcfc"
+                    class="amp-wp-f94bb8b">state of the web</span><span data-amp-original-style="color: #fcfcfc"
+                                                                        class="amp-wp-4a61a81"> report</span></p></div>
```

Comment:
Unfortunately you can't break `<p>` elements arbitrarily like this or it will insert new lines into the rendered HTML:

![image](https://user-images.githubusercontent.com/10931297/103704751-b0d4d800-4fa1-11eb-99c0-4052c31384ca.png)

![image](https://user-images.githubusercontent.com/10931297/103704786-c1854e00-4fa1-11eb-988b-e7958148d09a.png)

When I formatted the HTML for the first story I had to similarly compare it to the correctly rendering version and clean up the formatted HTML by collapsing back `<p>` elements and then inserting a few `<br>` where we actually wanted line break. It's a bit painful but not sure if there's an easier way, because it really is nicer to have unminified HTML to be honest.

```suggestion
            <div id="el-cdb28853-0908-4d31-a1ed-10a6fb4954f4" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0" class="amp-wp-a1b70b8">
              <p class="fill amp-wp-ae8dc76" data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:-1.325088339222615% 0;font-family:'Lato',sans-serif;font-size:0.404531em;line-height:1.5;text-align:initial;padding:0% 0%;color:#000000"><span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81">HTTP Archive's annual</span><br><span data-amp-original-style="font-weight: 900; color: #fcfcfc" class="amp-wp-f94bb8b">state of the web</span><span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81"> report</span>
              </p>
            </div>
```

Reply:
Yeah `white-space:pre-wrap` is a pain when you'd want nicely formatted output.

I manually went through the file & story now to fix any remaining issues, ensuring visual parity with the original, minified file.